tests/numpy/test_stack.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+
+# pylint: disable=import-outside-toplevel
+
+def try_multiple_nd(n):
+    import nums.numpy as nps
+
+    scalar = 1
+    tuple_1d = (1, 2)
+    list_1d = [1, 2]
+    np_2d = np.array([[1, 2]])
+    np_3d = np.array([[[1, 2]]])
+    nps_3d = nps.array([[[1, 2]]])
+    list_4d = [[[[1, 2]]]]
+
+    if n == 1:
```

Comment:
Can we simplify this code by passing the np and nps functions to compare?

Reply:
Yes.

Reply:
Yes.